luaui/Widgets/gui_gridmenu.lua

```diff
@@ -1103,7 +1113,7 @@ local function drawCategoryButtons()
 			drawnHoveredCat = cat
 		end
 
-		local catKey = Cfgs.keyLayouts[currentLayout][1][catIndex]
+		local catKey = Cfgs.keyLayouts[currentLayout]['category'] and Cfgs.keyLayouts[currentLayout]['category'][catIndex] or Cfgs.keyLayouts[currentLayout][1][catIndex]
```

Comment:
It seems this might not work well with non custom layouts and custom categories. You can either commit as it is since it satisfies your use case or make it a separate variable as to decouple keyLayout from custom categories as you can have a standard keyLayout without custom category keys

Reply:
Sorry, I don't quite understand what you mean? *IF* the `category` sub-table exists, it is used, but if it doesn't the default (row 1) is used.

Reply:
It's ok, current is good enough, thanks!